app/workers/detect_invariants.rb

```diff
@@ -0,0 +1,27 @@
+# Detect state that *should* be impossible in the system and report them to Sentry
+class DetectInvariants
+  include Sidekiq::Worker
+
+  def perform
+    detect_unprocessed_feedbacks
```

Comment:
could this method have a better name? `detect_application_choices_stuck_in_awaiting_references_state`?